spec/services/bulk_update/placements/create_placement_rows_spec.rb

```diff
@@ -48,6 +48,22 @@ module Placements
             )
           end
         end
+
+        context "given a csv with urns set as `ADDED MANUALLY`" do
+          let(:trainees) { create_list(:trainee, 3, :trn_received, :with_manual_placements) }
+          let(:csv) { [] }
+          let(:generate_report) { ::Reports::BulkPlacementReport.new(csv, scope: trainees).generate_report }
+
+          before do
+            generate_report
+            service
+          end
+
+          it "does not create placement rows" do
+            expect(csv).not_to be_empty
```

Comment:
That kinda shows how poorly designed this bit is. :disappointed: 

https://github.com/DFE-Digital/register-trainee-teachers/blob/d662cda1127c64d5429645916c1045b4cd222428/app/services/csv_report_service.rb#L21-L27